src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternPreferences.java

```diff
@@ -51,4 +56,9 @@ public boolean isEnforceLegalKey() {
     }
 
     public String getDefaultBibtexKeyPattern() { return defaultBibtexKeyPattern;}
+
+
```

Comment:
Duplicate empty line. @koppor is this caused by our Eclipse Settings? I'm seeing that all the time.